pkg/tsdb/cloudwatch/mocks/list_metrics_service.go

```diff
@@ -27,8 +28,8 @@ func (a *ListMetricsServiceMock) GetDimensionKeysByNamespace(string) ([]string,
 	return args.Get(0).([]string), args.Error(1)
 }
 
-func (a *ListMetricsServiceMock) GetHardCodedDimensionKeysByNamespace(string) ([]string, error) {
```

Comment:
This method is moved from this interface and put in the hardcoded_metrics service.